CONTRIBUTING.md

```diff
@@ -0,0 +1,70 @@
+- The Cluster-Version-Operator(CVO) manages the lifecycle of the marketplace-operator along with other internal Openshift components. Before deleting the marketplace-operator deployment, insruct CVO to stop managing the marketplace-operator with `oc apply -f examples/cvo.override.yaml`. 
+- Delete the marketplace-operator deployment with `oc delete deployment openshift-marketplace -n marketplace-operator`.
+- Install [Operator-SDK](https://github.com/operator-framework/operator-sdk).
+- `operator-sdk up local --namespace=openshift-marketplace --kubeconfig=<path-to-kubeconfig-file>` to complie the marketplace-operator and start the operator in your dev environment. 
+
+### Testing changes to RBAC policy
+- If there's any modification in existing RBAC poilicies, the modifications need to be tested using a separate deployment. Easiest way to do this is to do a local `operator-sdk build` using [this dockerfile](./Dockerfile) and then `docker push` the image to a registry. Replace the [image here](https://github.com/operator-framework/operator-marketplace/blob/master/deploy/operator.yaml#L23) with the newly built image, then do a `oc apply -f deploy/operator.yaml` to reinstall the operator.  
+
+## Marketplace-operator on vanilla Kubernetes
+
+- To set up marketplace-operator on vanilla Kubernetes, [install Operator-Lifecycle-Manager(OLM)](https://github.com/operator-framework/operator-lifecycle-manager/blob/master/Documentation/install/install.md) in a Kubernetes Cluster. 
+- `cd` into the marketplace-operator repository and install the marketplace-operator using `kubectl apply -f deploy/upstream`.
+- To delete the marketplace-operator deployment use `oc delete deployment marketplace`. 
+
+### Testing changes to RBAC policy
+- If there's any modification in existing RBAC poilicies, the modifications need to be tested using a seperate deployment. Easiest way to do this is to do a local `operator-sdk build` [this dockerfile](./Dockerfile) and then `docker push` the image to a registry. Replace the [image here](https://github.com/operator-framework/operator-marketplace/blob/master/deploy/upstream/07_operator.yaml#L28) with the newly built image, then do a `kubectl apply -f deploy/upstream/07_operator.yaml`.
+
+## Reporting bugs and creating issues
+
+If any part of the operator-marketplace project has bugs or documentation mistakes, please let us know by [opening an issue](https://github.com/operator-framework/operator-marketplace/issues/new) or a PR. 
+
+## Contribution flow
+
+This is an outline of what a contributor's workflow looks like:
+
+- Create a topic branch from where to base the contribution. This is usually master.
+- Make commits of logical units.
+- Make sure commit messages are in the proper format (see below).
+- Push changes in a topic branch to a personal fork of the repository.
```

Comment:
Shouldn't the first step be to for the repo?

Reply:
@aravindhp could you repeat this.

Reply:
Sorry, shouldn't forking the repo be the first step?

Reply:
@aravindhp I think I have listed forking the repo as the first step. Please let me know if you can see otherwise.

Reply:
You have in `Setting up a dev environment`. Here you just need to mention that you are creating a topic branch in the forked repo in your first bullet.

Reply:
@aravindhp Ah right. Added that now.